src/raylib/core/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(SRC
     ${SRCROOT}/Window.hpp
     ${SRCROOT}/Mouse.hpp
     ${SRCROOT}/Mouse.cpp
+    ${SRCROOT}/Scopes.hpp
```

Comment:
Wouldn't `scoped` be a better name for the name of this namespace and file.
Moreover, should files that does not describe a single class have an UpperCamelCase name? the name `Scopes.cpp` suggests that there is a class named `Scopes` available.